lib/internal/errors.js

```diff
@@ -1097,6 +1097,10 @@ E('ERR_NO_CRYPTO',
   'Node.js is not compiled with OpenSSL crypto support', Error);
 E('ERR_NO_ICU',
   '%s is not supported on Node.js compiled without ICU', TypeError);
+E('ERR_OPTION_INCOMPATIBLE_WITH_PLATFORM',
+  (option) => 'The option ' + option +
+    'is incompatible with the current platform' +
+    ',which is being used to run Node.js', TypeError);
```

Comment:
I think you’d want to adopt the text here if the name of the option changes, but just so you know, this would currently have missing spaces before `is incompatible`, after `platform`, and before `which`.

Reply:
thanks, will be fixed.

Reply:
fixed